Cargo.toml

```diff
@@ -36,6 +36,7 @@ glutin = "0.7"
 imagefmt = "4.0"
 specs = "0.7"
 wavefront_obj = "5.0"
+fnv = "1.0.5"
```

Comment:
Please try to keep the dependencies in alphabetical order. Also, I would recommend omitting the `.5` at the end so Cargo can automatically update patch releases.